examples/release_highlights/plot_release_highlights_1_0.py

```diff
@@ -0,0 +1,184 @@
+# flake8: noqa
+"""
+=======================================
+Release Highlights for scikit-learn 1.0
+=======================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.0! The library has
+been stable for quite some time, releasing version 1.0 is recognizing that and
+signalling it to our users. The release does not include any breaking changes,
+and as usual, and we do our best to follow a two-release deprecation cycles for
+any breaking changes.
```

Comment:
```suggestion
signalling it to our users. This release does not include any breaking changes
apart from the usual two-release deprecation cycle. For the future, we do our
best to keep this pattern.
```
Those are difficult sentences that long for precision. Maybe you have a better formulation.